src/makeCalculator.js

```diff
@@ -4,7 +4,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(num) {
+      calculator.result += num;
+    },
+    subtract(num) {
+      calculator.result -= num;
+    },
+    multiply(num) {
+      calculator.result *= num;
+    },
+    divide(num) {
+      calculator.result /= num;
+    },
```

Comment:
[CODE KNOWLEDGE] - When defining methods in an object, you can omit the function keyword for a more concise syntax. For example, instead of 'add: function(num) {...}', you can simply use 'add(num) {...}'.